scalar.c

```diff
@@ -230,6 +230,31 @@ static int set_recommended_config(int reconfigure)
 				     fsmonitor.key, fsmonitor.value);
 	}
 
+	/*
+	 * Set HTTP/1.1 for Azure DevOps URLs
+	 * We check for dev.azure.com and .visualstudio.com/ patterns
+	 * which are sufficient to identify ADO URLs
+	 */
+	if (!git_config_get_string("remote.origin.url", &value)) {
+		if (starts_with(value, "https://dev.azure.com") ||
```

Comment:
This may be incorrect, as some users use `<orgname>@dev.azure.com` and so we should consider a string search for `dev.azure.com/` in the URL, similar to how we check for `.visualstudio.com`.

Reply:
Right, a better approach might be to do `starts_with(value, "https://dev.azure.com/") || strstr(value, "@dev.azure.com/")`.